CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -440,15 +418,18 @@ import Vision
             return
         }
         
+        let testImgArr: [CGImage?] = self.testingImagesSizeCheck()
+        
         // we allow apps that integrate to supply their own sequence of images
         // for use in testing
-        let image = self.testingImageDataSource?.nextImage() ?? squareCardImage
+        let squareCardImg: CGImage  = testImgArr[0] ?? squareCardImage
```

Comment:
it's more swift-y to name this variable squareCardImage (the swift people usually don't want you to use abbreviations)